pom.xml

```diff
@@ -330,6 +330,11 @@
             <version>20.0</version>
         </dependency>
         <dependency>
+            <groupId>com.google.cloud</groupId>
+            <artifactId>google-cloud-storage</artifactId>
+            <version>1.2.1</version>
```

Comment:
At the final API review it was decided to make this a hard dependency. If in the future we change our mind about that decision, all we have to do is to add `<scope>provided</scope>` to the above config.